packages/api-client-core/src/operationBuilders.ts

```diff
@@ -115,7 +116,7 @@ export const actionOperation = (
 
   const selection = options?.select || defaultSelection;
   if (selection) {
-    actionOperation.fields!.push({ [modelApiIdentifier]: fieldSelectionToGQLBuilderFields(selection) });
+    actionOperation.fields!.push({ [modelSelectionField]: fieldSelectionToGQLBuilderFields(selection) });
```

Comment:
Important bit is here for bulk actions: as pointed out in the PR description, the field would be "widgets", but we need to get hydrations (below) for "widget".